crates/bcr-wdc-ebpp/src/onchain.rs

```diff
@@ -216,6 +211,9 @@ where
                 let total: btc::Amount = wlt
                     .list_unspent()
                     .filter(|output| !output.is_spent)
+                        .filter(|output| {
```

Comment:
hm why is this indented more? :thinking: 

Plus, the two filters could be combined, but I feel like it's more explici this way and afair the compiler makes one optimized loop out of it anyway